### PR TITLE
Disable the SSR build during dev build

### LINKED ISSUE #104: `febs` no longer recognizes `devServer` config options
Since the update to use `MultiCompiler`, the `runDevServer` option no longer parses out dev server options from the compiled instance.
Line 22 of `dev-server.js` has
```js
  const devServerOverrides = compiler.options ? compiler.options.devServer : null;
```
when the parameter `compiler` is now a `MultiCompiler` instance.
So for the same config that existed before that was accessed through `compiler.options.devServer` is now only available through `compiler.compiler[0].options.devServer`

### PR DESCRIPTION
**Pull request by peripateticus**
- Disable the SSR build when running `febs dev`.
- Fix #104 

Some background:
The SSR build adds significant amount of build time consequently slowing down live-reloading considerably. This disables the SSR build by default when running `dev` build.

### CODE REVIEW

**Comment by peripateticus** [comment]
I still need to update the docs, stay tuned...

**Comment by peripateticus** [comment]
This fixes #100 .

**Comment by peripateticus** [comment]
Readme updated, good to go.

File: lib/dev-server.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const runDevServer = (webpackDevServer, compiler) => {
 
   const port = 8080;
   const projectPath = process.cwd();
-  const devServerOverrides = compiler.options ? compiler.options.devServer : null;
+  const devServerOverrides = compiler.options ? compiler.compiler[0].options.devServer : null;
```
Comment (zcrumbo) [inline lib/dev-server.js]: If this is always a `MultiCompiler` instance I don't think this code will ever return true - isn't the structure 
```
 compiler: { compilers: [...] ...} 
```

Comment (peripateticus) [reply lib/dev-server.js]: Good catch, yep, `compiler.compilers[0]...`.

File: ReadMe.md [hunk 2]
```diff
@@ -156,6 +156,17 @@ Notes:
...
+#### `ssr` property
+
+This value determines whether or not to build the `vue-ssr-server-bundle.json` required
+for server-side rendering.
+
+Notes:
+- Enabling this will add additional time to the build as it essentially needs to run 2 builds, one
+for the client and one for the server. Additionally, the generated  `vue-ssr-server-bundle.json` is very large.
+- This is automatically disabled during development builds, i.e., `febs dev`.
+- If, for some reason, you do need this enabled during `dev` build, set `resolve.symlinks` property in `webpack.overrides.conf.js`
```
Comment (cowills) [inline ReadMe.md]: is `resolve.symlinks` correct here? i don't understand the connection to ssr

Comment (peripateticus) [reply ReadMe.md]: Good catch, I somehow conflated 2 completely separate issues!

**Pull request merged**